<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contactDataDialog</class>
 <widget class="QDialog" name="contactDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>313</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>313</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Contact numbers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1">
    <widget class="QLineEdit" name="protocolSmsEdit">
     <property name="statusTip">
      <string>Phone number for sending &quot;Protocol&quot;  SMS</string>
     </property>
     <property name="whatsThis">
      <string>Phone number for sending &quot;Protocol&quot;  SMS.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="alarmSmsEdit">
     <property name="statusTip">
      <string>Phone number for sending &quot;Alarm&quot;  SMS</string>
     </property>
     <property name="whatsThis">
      <string>Phone number for sending &quot;Alarm&quot;  SMS.</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QWidget" name="buttonsWrapper" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="confirmationButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="startSmsEdit">
     <property name="statusTip">
      <string>Phone number for sending &quot;Start&quot;  SMS</string>
     </property>
     <property name="whatsThis">
      <string>Phone number for sending &quot;Start&quot;  SMS.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="startSmsLabel">
     <property name="text">
      <string>Start SMS number: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="statusSmsLabel">
     <property name="text">
      <string>Status SMS number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="statusSmsEdit">
     <property name="statusTip">
      <string>Phone number for sending &quot;Status&quot;  SMS</string>
     </property>
     <property name="whatsThis">
      <string>Phone number for sending &quot;Status&quot;  SMS.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="alarmSmsNumber">
     <property name="text">
      <string>Alarm SMS number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="callNumEdit2">
     <property name="whatsThis">
      <string>Contact phone number</string>
     </property>
     <property name="accessibleName">
      <string>Contact phone number.</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="callNumEdit1">
     <property name="whatsThis">
      <string>Contact phone number</string>
     </property>
     <property name="accessibleName">
      <string>Contact phone number.</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="protocolSmsNumber">
     <property name="text">
      <string>Protocol SMS number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="callNumEdit3">
     <property name="whatsThis">
      <string>Contact phone number</string>
     </property>
     <property name="accessibleName">
      <string>Contact phone number.</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="callNumEdit4">
     <property name="whatsThis">
      <string>Contact phone number</string>
     </property>
     <property name="accessibleName">
      <string>Contact phone number.</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="callNumLabel1">
     <property name="text">
      <string>Call number 1:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="callNumLabel2">
     <property name="text">
      <string>Call number 2:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="callNumLabel3">
     <property name="text">
      <string>Call number 3:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="callNumLabel4">
     <property name="text">
      <string>Call number 4:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>startSmsEdit</tabstop>
  <tabstop>statusSmsEdit</tabstop>
  <tabstop>alarmSmsEdit</tabstop>
  <tabstop>protocolSmsEdit</tabstop>
  <tabstop>callNumEdit1</tabstop>
  <tabstop>callNumEdit2</tabstop>
  <tabstop>callNumEdit3</tabstop>
  <tabstop>callNumEdit4</tabstop>
  <tabstop>confirmationButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
